files/default/neutron-ha-tool.py

```diff
@@ -252,16 +252,19 @@ def run(args):
 
     elif args.l3_agent_migrate:
         LOG.info("Performing L3 Agent Migration for Offline L3 Agents")
+        # Move all routers by not filtering them
+        router_filter = NullRouterFilter()
         errors = retry_with_backoff(l3_agent_migrate, args)(
-            qclient, agent_picker, args.noop, args.now,
+            qclient, agent_picker, router_filter, args.noop, args.now,
             args.wait_for_router, args.ssh_delete_namespace)
 
     elif args.l3_agent_evacuate:
         LOG.info("Performing L3 Agent Evacuation from host %s",
                  args.l3_agent_evacuate)
+        router_filter = NullRouterFilter()  # TODO: Make this configurable
         errors = retry_with_backoff(l3_agent_evacuate, args)(
-            qclient, args.l3_agent_evacuate, agent_picker, args.noop,
-            args.wait_for_router, args.ssh_delete_namespace)
+            qclient, args.l3_agent_evacuate, agent_picker, router_filter,
```

Comment:
I agree with that, and the globals were an attempt to avoid that, let me think about how can we fix that.